frontend/src/scenes/organization/CreateOrganizationModal.tsx

```diff
@@ -9,23 +9,28 @@ export function CreateOrganizationModal({
     setIsVisible,
 }: {
     isVisible: boolean
-    setIsVisible: Dispatch<SetStateAction<boolean>>
+    setIsVisible?: Dispatch<SetStateAction<boolean>>
 }): JSX.Element {
     const { createOrganization } = useActions(organizationLogic)
     const [errorMessage, setErrorMessage] = useState<string | null>(null)
     const inputRef = useRef<Input | null>(null)
 
-    const closeModal: () => void = useCallback(() => {
-        setErrorMessage(null)
-        setIsVisible(false)
-        if (inputRef.current) inputRef.current.setValue('')
-    }, [inputRef, setIsVisible])
+    const closeModal: () => void = useCallback(
```

Comment:
What is this trying to achieve? This seems way too complex and possibly broken from how I understand `useCallback` works, so wondering what's going on here.

Reply:
Yeah, better to just have `if (setIsVisible)` inside the function instead of using some no-op.